doc/source/release-notes/Percona-XtraDB-Cluster-5.7.20-29.24.rst

```diff
@@ -0,0 +1,121 @@
+.. rn:: 5.7.20-29.24
+
+=====================================
+Percona XtraDB Cluster 5.7.20-29.24
+=====================================
+
+Percona is glad to announce the release of
+Percona XtraDB Cluster 5.7.20-29.24 on January 26, 2018.
+Binaries are available from the `downloads section
+<http://www.percona.com/downloads/Percona-XtraDB-Cluster-57/>`_
+or from our :ref:`software repositories <install>`.
+
+.. note:: Due to new package dependency,
+   Ubuntu/Debian users should use ``apt-get dist-upgrade``, ``apt upgrade``,
+   or ``apt-get install percona-xtradb-cluster-57`` to upgrade.
+
+Percona XtraDB Cluster 5.7.20-29.24 is now the current release,
+based on the following:
+
+* `Percona Server 5.7.20 <http://www.percona.com/doc/percona-server/5.7/release-notes/Percona-Server-5.7.20-17.html>`_
```

Comment:
It's based on https://www.percona.com/doc/percona-server/5.7/release-notes/Percona-Server-5.7.20-18.html there was no 5.7.20-17.